docs/source/azure/configuration/monitoring.rst

```diff
@@ -42,3 +42,34 @@ specified when running Cloud Custodian:
     .. code-block:: sh
 
         custodian run --metrics azure://<resource_group_name>/<app_insights_name>
+
+Blob Storage Output
```

Comment:
This section shows up by itself in the nav and looks correct, but it is a little out of place under the App Insights/metrics heading in the file.  This output is referring to the output of the app being able to go to disk or blob storage and isn't really metrics.   Since it is already has a nav heading like App Insights should it just be its own file? 

Or this file could become "Logging, Metrics and Output".

Also, if we don't like "Blob Storage Output" we could do something like "Writing to Azure Storage".

Reply:
I like "Writing to Azure Storage"